### ISSUE TITLE
A11y keyboard control Tab component TabList Tab navigation issue

### ISSUE THREAD

**Issue by DaveBest99** [angular/components#3422]
#### Bug, feature request, or proposal:
Bug: Screen reader keyboard navigation issue.

#### What is the expected behavior?
The following key commands should be used for navigating the TabList:
- With focus on a Tab element, pressing the Left arrow will move focus to the previous Tab in the TabList (and activate it if AutoFocus is enabled). Pressing the Left arrow when the focus is on the first Tab element in the TabList will move focus to the last Tab element in the list. Note, in order to support vertical tabs, Left arrow moves focus up and wraps around to the bottom.
- With focus on a Tab element, pressing the Right arrow will move focus to the next Tab element in the TabList (and activate it if AutoFocus is enabled). Pressing the Right arrow when the focus is on the last Tab element in the TabList will move focus to the first Tab element in the list. Note, in order to support vertical tabs, Right arrow moves focus down and wraps around to the top.
- Pressing the Home key moves focus to the first Tab element in the list.
- Pressing the End key moves focus to the last Tab element in the list.
- Optionally, pressing the Shift+F10  on a Tab element will open the pop-up menu if one is available.
- Optionally, if deletion is allowed, pressing the Delete key closes the current Tab element and its associated TabPanel.

#### What is the current behavior?
Currently arrow keys do not function as expected, and Home/End keys do not function at all.

#### Which versions of Angular, Material, OS, browsers are affected?
Testing was performed on Windows10 with JAWS17 and Firefox/IE11/Chrome browsers.

#### Is there anything else we should know?
A11y issue:
Left and Right Arrow keys moves cursor focus between the TabList elements, but does not wrap around. The Home and End keys do not function at all. As focus moves from Tab to Tab the Tab text is spoken by JAWS and shown on the braille display.
Note, some Tab components may be complex with lots of content, and would best be displayed in a separate modal window. This would allow for a clearly defined Heading hierarchy and region Landmark content structure within the TabPanel section. A Close button, or Escape key, could be used to close the Tab component. An alternative to providing a keystroke to close a Tab is to provide a context menu that is associated with the tab label. When focus is on the tab, pressing Shift+F10 or pressing the right mouse button will open a context menu with a Close button.
